docs/porting/security/psa_targets_in_mbed-os.md

```diff
@@ -0,0 +1,489 @@
+
+### Building TF-M and running regression tests
+
+To build TF-M and regression tests:
+
+1. Clone the [mbed-os-tf-m-regression-tests](https://github.com/ARMmbed/mbed-os-tf-m-regression-tests) repository.
+1. Update `mbed-os.lib` to the copy of Mbed OS that contains new target support, including the `targets.json` changes described in this document.
+1. Run:
+
+    ```
+    mbed deploy
+    ```
+
+1. Run:
+
+    ```
+    python3 build_tfm.py -m <new target> -t <toolchain> -c ConfigRegressionIPC.cmake
+    ```
+
+    The command builds a TF-M binary the service necessary to run regression tests.
+
+    <span class="notes">**Note:** Use this `ConfigRegressionIPC.cmake` only to run regression tests. The secure binary copied to Mbed OS (when you are ready to merge your new target) must be generated using the config option `ConfigCoreIPC.cmake`.</span>
+
+1. Ensure the TF-M binary was copied to the location defined by the target attribute `tfm_delivery_dir`. If no binary was created or copied, please raise an [issue on the Mbed OS repository](https://github.com/ARMmbed/mbed-os).
+
+1. To build Mbed OS and run regression tests, run:
+
+    ```
+    mbed compile -m <new target> -t <toolchain>
+    ```
+1. Flash the regression tests binary to the target and ensure all regression tests pass.
+
+### Building TF-M and running PSA compliance tests
+
+To build TF-M and compliance tests:
+
+1. Switch to the `mbed-os-tf-m-regression-tests` directory.
+1. Edit `main.cpp` and set:
+
+    - `RUN_PSA_COMPLIANCE_TESTS` to `1`.
+    - `RUN_REGRESSION_TESTS` to `0`.
+1. Run:
+
+    ```
+    python3 build_psa_compliance.py -m <new target> -t <toolchain> -s CRYPTO
+    ```
+
+    The command builds PSA API compliance tests for the crypto suite.
+1. Run:
+
+   ```
+   python3 build_tfm.py -m <new target> -t <toolchain> -c configs/ConfigPsaApiTestIPC.cmake -s CRYPTO
+   ```
+
+   The command builds TF-M including PSA API compliance tests.
+1. To build Mbed OS and run PSA API compliance tests, run:
+
+    ```
+    mbed compile -m <new target> -t <toolchain>
+    ```
+1. Flash the PSA API compliance tests binary to the target and ensure all PSA API compliance tests pass.
+
+    <span class="notes">**Note:** Any PSA API compliance tests that fail in the TF-M example application will fail in Mbed OS as well.</span>
+
+### Building TF-M and creating an Mbed OS pull request
+
+<span class="notes">**Note:** Please ensure that both regression tests and PSA API compliance tests pass before creating an Mbed OS pull request.</span>
+
+To build TF-M and create an Mbed OS pull request:
+
+1. Switch to the `mbed-os-tf-m-regression-tests` directory.
+1. Run:
+
+    ```
+    python3 build_tfm.py -m <new target> -t <toolchain> --commit
+    ```
+
+    The command builds TF-M with the config `ConfigCoreIPC.cmake` ,copies the TF-M binary to the location defined by the target attribute `tfm_delivery_dir`, and commits the changes to Mbed OS.
```

Comment:
Should we make this change? or is the space where it is because of the particular markdown tool we are using?
```diff
- the config `ConfigCoreIPC.cmake` ,copies the
+ the config `ConfigCoreIPC.cmake`, copies the
```

Reply:
It's just a typo

Reply:
fixed